news/4287.feature

```diff
@@ -0,0 +1 @@
+Slug based linked headlines in volto-slate: [tiberiuichim , nileshgulia1]
```

Comment:
```suggestion
Added slug-based linked headlines in `volto-slate`. @tiberiuichim, @nileshgulia1
```

Reply:
Fixed in [de9e57f](https://github.com/plone/volto/pull/4287/commits/de9e57f3797e705a451351046a9688722a28500c)